### ISSUE TITLE
Rule SA1008 (OpeningParenthesisMustBeSpacedCorrectly) violation for interfaces methods returning tuples

### ISSUE THREAD

**Issue by BennieCopeland** [DotNetAnalyzers/StyleCopAnalyzers#2526]
I am receiving a warning for code similar to the following:

```csharp
interface SomeInterface
{
    (bool Value1, object Value2) GetSomeValue();
}
```

**Comment by sharwell** [comment]
@BennieCopeland What version of StyleCop Analyzers are you using? This appears to be a duplicate of #2308, which is fixed in the latest 1.1.0 beta releases.
